code/modules/admin/admin_verbs.dm

```diff
@@ -296,6 +296,8 @@ GLOBAL_LIST_INIT(admin_verbs_maintainer, list(
 			verbs += /client/proc/ss_breakdown
 			verbs += /client/proc/show_gc_queues
 			verbs += /client/proc/toggle_mctabs
+			verbs += /client/proc/debug_timers
+			verbs += /client/proc/timer_log
```

Comment:
There's a comment above about making a viewruntimes list -- might be time to bite the bullet and do that if you don't mind.

Reply:
I have a PR open which already does that, I can just add these verbs when that PR is merged since this is probably gonna get merged first